Xamarin.Forms.Core/BindablePropertyConverter.cs

```diff
@@ -102,5 +106,17 @@ BindableProperty ConvertFrom(Type type, string propertyName, IXmlLineInfo linein
 				throw new XamlParseException($"The PropertyName of {type.Name}.{name} is not {propertyName}", lineinfo);
 			return bp;
 		}
+
+		Type FindTypeForVisualState(List<object> parentObjects)
+		{
+			var obj = parentObjects.Skip(3).Take(1).FirstOrDefault();  // Skip this Setter, VisualState, and VisualStateGroup
```

Comment:
you should skip them one by one, and verify their types

Reply:
Added type verification.